### PR TITLE
SDI-4: ✨ Add production environment

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -2,12 +2,9 @@
 # Per environment values which override defaults in hmpps-nomis-prisoner-api/values.yaml
 
 generic-service:
-  replicaCount: 4
```
Comment (andymarke) [inline helm_deploy/values-preprod.yaml:5]: I assume 4 is défault anyway?

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+# Per environment values which override defaults in hmpps-nomis-prisoner-api/values.yaml
+
+generic-service:
+  ingress:
+    host: nomis-prisoner.aks-live-1.studio-hosting.service.justice.gov.uk
```
Comment (andymarke) [inline helm_deploy/values-prod.yaml:6]: oooh - "prisoner" not "prsner" - I bet correct the nomis update service

**Review by andymarke — APPROVED** [review]
LGTM

**Review by steverendell — APPROVED** [review]

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (petergphillips) [reply helm_deploy/values-preprod.yaml:5]: yes, that value is in `values.yaml` too so doesn't need repeating here

**Pull request merged**